logger.go

```diff
@@ -28,7 +28,7 @@ var defaultLogger = func(lvl logging.Level, s string, args ...interface{}) {
 		Log(s, args...)
 		return
 	}
-	if !Debug && lvl > logging.Debug {
+	if !Debug && lvl <= logging.Debug {
```

Comment:
I'm not sure about this changement the problem should be somewhere else because we have: 
SQL = 0
Debug = 1
Info = 2
Warn = 3 

so the lvl > logging.Debug sounds right.

I would also suggest to do it in an other way (but I'm not sure of the impact (except for readability) :

```go
    switch lvl {
    case logging.SQL:
        if len(args) > 0 {
			xargs := make([]string, len(args))
			for i, a := range args {
				switch a.(type) {
				case string:
					xargs[i] = fmt.Sprintf("%q", a)
				default:
					xargs[i] = fmt.Sprintf("%v", a)
				}
			}
			s = fmt.Sprintf("%s - %s | %s", lvl, s, xargs)
                } else {
                        s = fmt.Sprintf("%s - %s", lvl, s)
                }
    default:
        if !Debug && lvl <> logging.Debug { 
                return
        }
        s = fmt.Sprintf(s, args...)
	s = fmt.Sprintf("%s - %s", lvl, s)
    }
```

Reply:
It's not that easy to read, but the new condition is correct. We need to trim logs from levels lower or equal than debug, in non-debug mode.

It's a simple boolean logic mistake.